provelo_custom_invoice_auto_open/models/sale_order.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.multi
+    def action_invoice_create(self, grouped=False, final=False):
+        invoice_ids = super().action_invoice_create(grouped=grouped, final=final)
+        account_invoice_auto_open_group_id = self.env.ref(
+            "provelo_custom_invoice_auto_open.account_invoice_auto_open_group"
+        )
+        if account_invoice_auto_open_group_id.id in self.env.user.groups_id.ids:
```

Comment:
why not:
```suggestion
        if self.env.user.has_group(
            "provelo_custom_invoice_auto_open.account_invoice_auto_open_group"
        ):
```
as you suggested in the task?

Reply:
indeed why ? fixed

![](https://media.giphy.com/media/gxSc4hTA9QhsFagSbs/giphy.gif)